docs/en/action-menu-items/custom-browser-protocols.md

```diff
@@ -250,6 +249,18 @@ The data flow looks like this: once you click the AMI in {% include product %},
 
 {% include info title="Info" content="For additional information on troubleshooting with Monterey, [visit this community post](https://community.shotgridsoftware.com/t/amis-stopped-working-on-osx-monterey/16886)." %}
 
+**Debugging Tip:** Catching an error and then displaying it in a popup will allow you to see if there were any issues executing the Python script without it erroring silently. Here is an example snippet that can be added to your Python script for a try on error:
+
+```python
```

Comment:
This is actually AppleScript, not Python (sorry I should have been clearer in my guidance on the ticket). The first sentence is fine because it will help display any issues that arise from the executed Python script. The second sentence needs to change to `... to your AppleScript for a ...`. Line 254 should also change to `applescript`.

Also I don't mind it being a new section in the doc here, but it is a modification of the apple script provided on line 175.

Reply:
Thanks Phil! I like your suggestions. Because I thought it was a python snippet, I placed it at the bottom of the section. But now that I know it's an applescript, I've moved it up. Thanks again!